tests/test_methods_and_attributes.py

```diff
@@ -404,6 +404,19 @@ def test_accepts_none(msg):
     assert m.ok_none4(None) == -1
     assert m.ok_none5(None) == -1
 
+    with pytest.raises(TypeError) as excinfo:
+        m.no_none_kw(None)
```

Comment:
`no_none_kw` seems to suggest that one keyword arguments with `None` as the value are forbidden. At least that's how I'm reading it.

Reply:
Also after the others, `no_none{1..5}` and `ok_none{1..5}`?
Any suggestions? `no_none_as_kw`?

Reply:
`no_none` sounds good.